domain/src/main/kotlin/app/discord/user/dto/UserRegisterEvent.kt

```diff
@@ -0,0 +1,11 @@
+package app.discord.user.dto
+
+import java.time.OffsetDateTime
+
+data class UserRegisterEvent(
+    val guildId: String,
+    val userId: String,
+    val username: String,
+    val globalName: String,
+    val registerTime: OffsetDateTime
+)
```

Comment:
마찬가지로 UserIdentifier 사용 고려해보시기 바랍니다.
```suggestion
data class UserRegisterEvent(
    val userIdentifier: UserIdentifier,
    val username: String,
    val globalName: String,
    val registerTime: OffsetDateTime
)
```